test/test_plot_results.jl

```diff
@@ -0,0 +1,68 @@
+IS.configure_logging(console_level = Logging.Info)
```

Comment:
You shouldn't reconfigure logging here.  If you want to enable Info prints then you should set the relevant environment variables.

Reply:
@daniel-thom I added this to all the tests that require Simulations. What's the alternative approach?

Reply:
@daniel-thom do you have an example of how to set the environmental variables? I'm not sure exactly what you mean

Reply:
The `run_tests` function in test/runtests.jl configures a global logger for all unit tests.  It sets the console logging to Error.  It sounds like you may just want to set that to Info.

If you don't want to make that change then you can set an environment variable to override the default.  However, this method doesn't work if you are running `Pkg.test()`.  It only works if you are doing `include(test/runtests.jl)` in the REPL.

```
ENV["SYS_CONSOLE_LOG_LEVEL"] = "Info"
include(test/runtests.jl)
```

Reference:  https://github.com/NREL/InfrastructureSystems.jl/blob/master/docs/src/man/tests.md